Example/GiniVision/SelectAPIViewController.swift

```diff
@@ -37,8 +37,7 @@ final class SelectAPIViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        let customClientId = UserDefaults.standard.string(forKey: kSettingsGiniSDKClientIdKey) ?? ""
-        let clientId = customClientId != "" ? customClientId : kGiniClientId
+        let clientId = ProcessInfo.processInfo.environment["client_id"] ?? ""
```

Comment:
What about the file? Why are you only reading from the ENV?

Reply:
Yes, I forgot about this one.